resources/contenttweaker/lang/en_us.lang

```diff
@@ -1 +1,5 @@
-item.contenttweaker.printed_parallel.name=Printed Parallel Circuit
\ No newline at end of file
+item.contenttweaker.printed_parallel.name=Printed Parallel Circuit
+item.contenttweaker.coin_bronze.name=Bronze Coin (Steam)
+item.contenttweaker.coin_steel.name=Steel Coin (LV)
+item.contenttweaker.coin_aluminium.name=Aluminium Coin (MV)
+item.contenttweaker.coin_stainless_steel.name=Stanless Steel Coin (HV)
```

Comment:
`item.contenttweaker.coin_stainless_steel.name=Stanless Steel Coin (HV)` should change to:
`item.contenttweaker.coin_stainless_steel.name=Stainless Steel Coin (HV)`